experimental/oauthtokenretriever/token.go

```diff
@@ -13,6 +13,15 @@ import (
 	"golang.org/x/oauth2/clientcredentials"
 )
 
+const (
+	AppURL        = "GF_APP_URL"
+	AppClientID   = "GF_PLUGIN_APP_CLIENT_ID"
+	AppPrivateKey = "GF_PLUGIN_APP_PRIVATE_KEY"
+	// nolint:gosec
+	// AppClientSecret constant represents a string index value for the secret, not the secret itself.
+	AppClientSecret = "GF_PLUGIN_APP_CLIENT_SECRET"
```

Comment:
These are needed so that the client side know what keys to set in the config when the request is created. Unless you had something else in mind?

Reply:
Yeah but they were unexported before. Grafana is not just repeating them:
https://github.com/grafana/grafana/blob/c69dd41bf9a88285dc43cea40c6e31b66914abb1/pkg/plugins/envvars/envvars.go#L52

The plan is to import those consts from `grafana-plugin-sdk-go` instead, right?

Reply:
>The plan is to import those consts from grafana-plugin-sdk-go instead, right?

Yeah exactly!

Reply:
Makes sense, but it could feel a bit weird since those are now under `experimental`. But it's Grafana core so it's probably okay :+1: 

I am making this as resolved then!